xpath.d.ts

```diff
@@ -1,11 +1,35 @@
 /// <reference lib="dom" />
 
-type SelectedValue = Node | Attr | string | number | boolean;
-interface XPathSelect {
-    (expression: string, node?: Node): Array<SelectedValue>;
-    (expression: string, node: Node, single: true): SelectedValue | undefined;
+export type SelectedValue = Node | string | number | boolean | null;
+
+export interface XPathSelect {
+    (expression: string, node: Node): Array<Node>;
```

Comment:
It looks like these three functions should have the same return types as the three corresponding `select` functions below.

Reply:
Yes, they do. That is because `useNamespaces` returns a function that follows this interface.

Reply:
Yes, that's my point. This .d.ts file has them returning different types, but it should have them returning the same types.

Reply:
I fixed them.